test/integration/merkle-lockup/factory/create-merkle-lockup-lt/createMerkleLockupLT.t.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.22 <0.9.0;
+
+import { ud2x18 } from "@prb/math/src/UD2x18.sol";
+
+import { Errors } from "src/libraries/Errors.sol";
+import { ISablierV2MerkleLockupLT } from "src/interfaces/ISablierV2MerkleLockupLT.sol";
+import { MerkleLockup, MerkleLockupLT } from "src/types/DataTypes.sol";
+
+import { MerkleLockup_Integration_Test } from "../../MerkleLockup.t.sol";
+
+contract CreateMerkleLockupLT_Integration_Test is MerkleLockup_Integration_Test {
+    function setUp() public override {
+        MerkleLockup_Integration_Test.setUp();
+    }
+
+    modifier whenTranchesPercentageSumIsNotOneHundred() {
```

Comment:
If we go with `unlockPercentage` then we can call it `whenUnlockPercentageSumisNot100`? Thats still too long 😅